README.md

```diff
@@ -207,17 +207,16 @@ First compile the cert-manager E2E test binary as follows:
 
 ```sh
 cd projects/cert-manager/cert-manager
-bazel build //test/e2e:e2e
+make e2e-build
 ```
 
 And then upload the binary to the remote VM and run them against cert-manager installed in the crc OpenShift cluster:
 
 ```sh
 cd projects/cert-manager/cert-manager-olm
 make crc-e2e \
-  OPENSHIFT_VERSION=4.8 \
-  PULL_SECRET=~/Downloads/pull-secret \
-  E2E_TEST=../cert-manager/bazel-bin/test/e2e/e2e.test
+  OPENSHIFT_VERSION=4.X \
+  E2E_TEST=../cert-manager/_bin/test/e2e.test
```

Comment:
This is the new path where the e2e.test binary will be created.
The PULL_SECRET is not needed because this target no longer creates the VM if not found.